nodes/BraveSearch/operations/web/index.ts

```diff
@@ -17,10 +18,4 @@ const details: INodePropertyOptions = {
 // All properties are only contextually visible for this operation
 parameters.forEach((p: INodeProperties) => (p.displayOptions = { show: { operation: [key] } }));
 
-// Simple implementation right now, but could do validation/etc. in future updates
-const buildQuery = (query: Record<string, any>) => {
-	const { query: q, ...rest } = query; // Destructure 'query' as 'q' and collect the rest
-	return { q, ...rest }; // Return the new object
-};
```

Comment:
As noted in the original code, each operation has its own `buildQuery` method, enabling us to later implement per-operation parameter-validation and more. Conversion of array values to CSVs is automatically handled by the request process (via Axios). I'd prefer to keep query-building/validation primarily within each operation's module, though we could certainly find value in extending a base implementation.